modules/quota-increase/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = ">= 0.12.26"
+}
+
+resource "aws_servicequotas_service_quota" "increase_quotas" {
+  for_each = var.resources_to_increase
+
+  quota_code = local.codes[each.key].quota_code
+  service_code = local.codes[each.key].service_code
+  value = each.value
+}
+
+locals {
+  codes = {
+    nacl_rules = {
+      quota_code = "L-2AEEBF1A"
+      service_code = "vpc"
+    },
+    nat_gateway = {
+      quota_code = "L-FE5A380F"
```

Comment:
NIT: add a comment explaining how to look these up. The comment could also say, "See the README for how to retrieve these values."